Utilities.ps1

```diff
@@ -79,13 +79,33 @@ function Get-ExchangeObjectLocation {
 
     $isLocal = $exchangeObjectTypeDetails -like '*Mailbox'
     $isRemote = $exchangeObjectTypeDetails -like 'Remote*'
-
-    if ($isLocal -and (-not $isRemote)) {
-        return [ExchangeObjectLocation]::exchangeOnPremises
-    } elseif ($isRemote) {
-        return [ExchangeObjectLocation]::exchangeOnline
+    $isMailUser = $exchangeObjectTypeDetails -eq 'MailUser'
+
+    $sessionScope = Get-PSSession |
+        Where-Object { $_.State -eq 'Opened' -and $_.ConfigurationName -eq 'Microsoft.Exchange' } |
+        Select-Object -ExpandProperty ComputerName
+    if ($sessionScope -eq 'outlook.office365.com') {
```

Comment:
Leave empty lines before and after control blocks

Reply:
Done